app/assets/stylesheets/provider/_tables.scss

```diff
@@ -134,9 +134,13 @@ table.data,
 table.list {
   td,
   th {
-    word-wrap: break-word;
     border-bottom: $border-width solid $border-color;
     font-weight: $font-weight-normal;
+    word-break: break-all;
+
+    &:nth-child(3) {
```

Comment:
This looks way too specific IMO (and a bit hack-ish).

Reply:
I think `word-wrap: anywhere;` is used for this very purpose in other classes.

Reply:
word-wrap is not working in FF

Reply:
True! I'm still not entirely convinced about nth-child. What about creating a class for the price?